index.html

```diff
@@ -74,8 +74,13 @@ <h1 class="toblur">
                 </ul>
             </div>
         </section>
+
+        <section class="modal">
+
+        </section>
+
         <section class="work-section">
-            <a id="portfolio" name="portfolio"></a>
+            <!--<a id="portfolio" name="portfolio"></a>
```

Comment:
- [ ] please try to remove all the unused commented code to keep your file clean and professional